src/components/atoms/TextField/index.jsx

```diff
@@ -20,25 +29,31 @@ const TextField = forwardRef((props, ref) => {
         id={id}
         ref={ref}
         type="text"
+        {...register(name)}
         {...otherInputProps}
       />
+      {errorMessage && <p>{errorMessage}</p>}
```

Comment:
Note for future reference with more advanced forms you can use the [error message](https://react-hook-form.com/api/useformstate/errormessage) component.

Reply:
@HcroakerDev Thanks for the suggestion, I've switched over to using the component in the latest commit. All tests pass :)